c2corg_ui/templates/outing/edit.html

```diff
@@ -81,22 +59,151 @@ <h2 class="heading show-phone">
+                  <span class="input-group-addon">m</span>
+                </div>
+                <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.length_total.$valid && outing.length_total"></span>
+                <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.length_total.$invalid && outing.length_total"></span>
+                <div class="help-block" ng-messages="editForm.length_total.$error">
+                  <p ng-message="number" class="label label-danger" style="{{ (editForm.length_total.$invalid && editForm.length_total.$touched) ? 'display:inline' : '' }}" translate>This field must be a number.</p>
+                </div>
+              </div>
+                            
+              ## PARTIAL TRIP
+              <div class="form-group data half" ng-class="{'has-success': outing.partial_trip }">
+                <label translate>partial_trip</label>
+                <select ng-model="outing.partial_trip" class="form-control">
+                  <option value="true">yes</option>
+                  <option value="false">no</option>
+                  <option>no info</option>
+                </select>
+              </div>
+
+              ## DURATION
+              <div class="form-group data half" ng-class="{'has-success': outing.duration}">
+                <label translate>duration</label>
+                <div class="input-group">
+                  <input type="number" min="1" minlength="1" name="duration" ng-model="outing.duration" class="form-control" />
+                  <span class="input-group-addon">min</span>
+                </div>
+              </div>
+              
+            ## ROUTE DESCRIPTION
+            <div class="data full form-group" ng-class="{'has-success': outing.route_description}">
+              <label translate>route_description</label>
+              <textarea class="form-control" ng-model="outing.locales[0]['conditions_levels']" placeholder="{{mainCtrl.translate('describe route_conditions')}}"></textarea>
+            </div>
+
+              ## MAP
+              <app-map app-map-edit-ctrl="editCtrl" class="map-edit"></app-map>
+              <div class="upload-file">
+                <button class="orange-btn btn" type="button" translate>upload a GPS track <span class="glyphicon glyphicon-upload"></span></button>
+                <input type="file" />
+              </div>
+
+          </div>
+        </section>
+        
+        <section class="section associations">
+          <h2 class="heading show-phone"><span translate>associations </span></h2>
+          <div class="content route-associations" id="title-edit">
+            <app-add-association parent-id="document.document_id" added-documents="added"></app-add-association>
+            <%include file="../document/associated_routes.html"/>
```

Comment:
Move to ``templates/routes/associated.html`` instead? Ditto for outings and waypoints.